corehq/apps/app_manager/static/app_manager/js/forms/advanced/actions.js

```diff
@@ -391,6 +391,39 @@ hqDefine('app_manager/js/forms/advanced/actions', function () {
                     return null;
 
                 });
+
+                self.moveable = ko.computed(function () {
```

Comment:
I'm not too familiar with this code, but how hard would it be to have subcases explicitly define which case tags they are the subcase of? Somewhat like Open Cases does. 
Probably out of scope, and might not be simpler since you'd still have to make sure you aren't rearranging subcases before their parents anyway.

Reply:
Yeah. I think that's out of scope for this work. it seems like we'd want solutions feedback to see if that's a change they'd even want?